src/WebApi/WebApi/Startup.cs

```diff
@@ -82,6 +73,14 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IApiVers
         Log.Debug("Setting UseRouting");
         app.UseRouting();
 
+        Log.Debug("Setting UseCors");
+        app.UseCors(builder =>
+        {
+            builder.WithOrigins(Configuration["AllowedOrigins"].Split(';'))
+                   .AllowAnyHeader()
+                   .AllowAnyMethod();
+        });
+
```

Comment:
Hey @TomasNiemczyk98, It looks great for a sunny day scenario, but I would argue that we need to update it a little bit to avoid possible issues.

I propose that you check if `AllowedOrigins` exist before you try to split it. I can imagine that if API and UI are running on the same domain the variable would be empty or deleted from the appsetting.json and that would cause a null reference exception.

**Optional:**I think moving this block of code to an extension inside WebApi/Modules would be great to keep the startup.cs clean.